packages/node-core/src/db/sync-helper.ts

```diff
@@ -72,12 +72,18 @@ function escapedName(...args: string[]): string {
   return args.map((a) => `"${a}"`).join('.');
 }
 
-function commentOn(type: 'CONSTRAINT' | 'TABLE' | 'COLUMN' | 'FUNCTION', entity: string, comment: string): string {
-  return `COMMENT ON ${type} ${entity} IS E'${comment}'`;
+function commentOn(
+  type: 'CONSTRAINT' | 'TABLE' | 'COLUMN' | 'FUNCTION',
+  entity: string,
+  comment: string,
+  constraint?: string
```

Comment:
I don't think this should live here. Keep the changes to `commentConstraintQuery`